--- gimageview-0.2.1/src/pixmaps/Makefile.am~	2004-07-16 06:10:01.000000000 +0000
+++ gimageview-0.2.1/src/pixmaps/Makefile.am	2004-07-16 06:10:01.000000000 +0000
@@ -1,4 +1,4 @@
-default_icondir = $(DESTDIR)$(gimv_icondir)/default
+default_icondir = $(gimv_icondir)/default
 
 default_icon_DATA =		\
 	alert.xpm		\
--- gimageview-0.2.1/src/pixmaps/Makefile.in~	2004-07-16 06:09:57.000000000 +0000
+++ gimageview-0.2.1/src/pixmaps/Makefile.in	2004-07-16 06:09:57.000000000 +0000
@@ -124,7 +124,7 @@
 localedir = @localedir@
 plugindir = @plugindir@
 
-default_icondir = $(DESTDIR)$(gimv_icondir)/default
+default_icondir = $(gimv_icondir)/default
 
 default_icon_DATA =  	alert.xpm			archive.xpm			back.xpm			close.xpm			dotfile.xpm			folder-go.xpm			folder-open.xpm			folder-link-open.xpm 		folder-up.xpm			folder.xpm 			folder-link.xpm 		folder48.xpm 			forward.xpm			fullscreen.xpm			gimageview.xpm			gimv_icon.xpm			image.xpm			home.xpm			leftarrow.xpm			nav_button.xpm 			nfolder.xpm			no_zoom.xpm			paper.xpm			prefs.xpm			question.xpm			refresh.xpm			resize.xpm			rightarrow.xpm			rotate.xpm			search.xpm			skip.xpm			small_archive.xpm		small_close.xpm			small_home.xpm			small_refresh.xpm		small_up.xpm			up.xpm				stop.xpm			up.xpm				zoom.xpm			zoom_fit.xpm			zoom_in.xpm			zoom_out.xpm			next_t.xpm			prev_t.xpm			ff.xpm				rw.xpm				play.xpm			pause.xpm			stop2.xpm			eject.xpm
 
--- gimageview-0.2.1/plugins/archiver/Makefile.am~	2004-07-16 07:19:09.000000000 +0000
+++ gimageview-0.2.1/plugins/archiver/Makefile.am	2004-07-16 07:19:09.000000000 +0000
@@ -4,7 +4,7 @@
 	libarc_tar_ext.la \
 	libarc_zip_ext.la
 
-archiver_plugindir = $(DESTDIR)$(plugindir)/$(ARCHIVER_PLUGIN_DIR)
+archiver_plugindir = $(plugindir)/$(ARCHIVER_PLUGIN_DIR)
 
 INCLUDES = -I$(top_srcdir)/src $(GTK_CFLAGS)
 
--- gimageview-0.2.1/plugins/archiver/Makefile.in~	2004-07-16 07:19:15.000000000 +0000
+++ gimageview-0.2.1/plugins/archiver/Makefile.in	2004-07-16 07:19:15.000000000 +0000
@@ -127,7 +127,7 @@
 archiver_plugin_LTLIBRARIES =  	libarc_lha_ext.la 	libarc_rar_ext.la 	libarc_tar_ext.la 	libarc_zip_ext.la
 
 
-archiver_plugindir = $(DESTDIR)$(plugindir)/$(ARCHIVER_PLUGIN_DIR)
+archiver_plugindir = $(plugindir)/$(ARCHIVER_PLUGIN_DIR)
 
 INCLUDES = -I$(top_srcdir)/src $(GTK_CFLAGS)
 
--- gimageview-0.2.1/plugins/image_saver/Makefile.am~	2004-07-16 07:19:37.000000000 +0000
+++ gimageview-0.2.1/plugins/image_saver/Makefile.am	2004-07-16 07:19:37.000000000 +0000
@@ -3,7 +3,7 @@
 	libpnm_saver.la \
 	libxvpics_saver.la
 
-image_saver_plugindir = $(DESTDIR)$(plugindir)/$(IMAGE_SAVER_PLUGIN_DIR)
+image_saver_plugindir = $(plugindir)/$(IMAGE_SAVER_PLUGIN_DIR)
 
 INCLUDES = -I$(top_srcdir)/src $(GTK_CFLAGS)
 
--- gimageview-0.2.1/plugins/image_saver/Makefile.in~	2004-07-16 07:19:54.000000000 +0000
+++ gimageview-0.2.1/plugins/image_saver/Makefile.in	2004-07-16 07:19:54.000000000 +0000
@@ -127,7 +127,7 @@
 image_saver_plugin_LTLIBRARIES =  	libpng_saver.la 	libpnm_saver.la 	libxvpics_saver.la
 
 
-image_saver_plugindir = $(DESTDIR)$(plugindir)/$(IMAGE_SAVER_PLUGIN_DIR)
+image_saver_plugindir = $(plugindir)/$(IMAGE_SAVER_PLUGIN_DIR)
 
 INCLUDES = -I$(top_srcdir)/src $(GTK_CFLAGS)
 
--- gimageview-0.2.1/plugins/thumbnail/Makefile.am~	2004-07-16 07:20:05.000000000 +0000
+++ gimageview-0.2.1/plugins/thumbnail/Makefile.am	2004-07-16 07:20:05.000000000 +0000
@@ -7,7 +7,7 @@
 	libee_thumb.la		\
 	libxvpics_thumb.la
 
-thumbnail_plugindir = $(DESTDIR)$(plugindir)/$(THUMBNAIL_PLUGIN_DIR)
+thumbnail_plugindir = $(plugindir)/$(THUMBNAIL_PLUGIN_DIR)
 
 INCLUDES = -I$(top_srcdir)/src $(GTK_CFLAGS)
 
--- gimageview-0.2.1/plugins/thumbnail/Makefile.in~	2004-07-16 07:20:10.000000000 +0000
+++ gimageview-0.2.1/plugins/thumbnail/Makefile.in	2004-07-16 07:20:10.000000000 +0000
@@ -127,7 +127,7 @@
 thumbnail_plugin_LTLIBRARIES =  	libgimv_thumb.la		libnautilus_thumb.la		libkonqueror_thumb.la		libkonq_old_thumb.la		libgqview_thumb.la		libee_thumb.la			libxvpics_thumb.la
 
 
-thumbnail_plugindir = $(DESTDIR)$(plugindir)/$(THUMBNAIL_PLUGIN_DIR)
+thumbnail_plugindir = $(plugindir)/$(THUMBNAIL_PLUGIN_DIR)
 
 INCLUDES = -I$(top_srcdir)/src $(GTK_CFLAGS)
 
--- gimageview-0.2.25/Makefile.am~	2004-07-16 07:56:16.000000000 +0000
+++ gimageview-0.2.25/Makefile.am	2004-07-16 07:56:16.000000000 +0000
@@ -4,7 +4,7 @@
 
 CLEANFILES = *~ .deps/* .*.swp .*.swo
 
-endocdir = $(DESTDIR)$(gimv_docdir)/text/C
+endocdir = $(gimv_docdir)/text/C
 endoc_DATA = \
 	AUTHORS \
 	COPYING \
--- gimageview-0.2.25/Makefile.in~	2004-07-16 07:56:36.000000000 +0000
+++ gimageview-0.2.25/Makefile.in	2004-07-16 07:56:36.000000000 +0000
@@ -205,7 +205,7 @@
 
 CLEANFILES = *~ .deps/* .*.swp .*.swo
 
-endocdir = $(DESTDIR)$(gimv_docdir)/text/C
+endocdir = $(gimv_docdir)/text/C
 endoc_DATA =  	AUTHORS 	COPYING 	ChangeLog 	README 	INSTALL 	NEWS 	TODO 	HACKING
 
 
--- gimageview-0.2.25/doc/ja/Makefile.am~	2004-07-16 07:57:17.000000000 +0000
+++ gimageview-0.2.25/doc/ja/Makefile.am	2004-07-16 07:57:17.000000000 +0000
@@ -1,4 +1,4 @@
-jahtmldir =  $(DESTDIR)$(gimv_docdir)/html/ja
+jahtmldir =  $(gimv_docdir)/html/ja
 jahtml_DATA = \
 	gimageview.css \
 	index.html \
@@ -16,7 +16,7 @@
 	document.html \
 	changelog.html
 
-jatextdir = $(DESTDIR)$(gimv_docdir)/text/ja
+jatextdir = $(gimv_docdir)/text/ja
 jatext_DATA = \
 	COPYING.ja \
 	ChangeLog.ja \
--- gimageview-0.2.25/doc/ja/Makefile.in~	2004-07-16 07:57:29.000000000 +0000
+++ gimageview-0.2.25/doc/ja/Makefile.in	2004-07-16 07:57:29.000000000 +0000
@@ -199,11 +199,11 @@
 localedir = @localedir@
 plugindir = @plugindir@
 
-jahtmldir = $(DESTDIR)$(gimv_docdir)/html/ja
+jahtmldir = $(gimv_docdir)/html/ja
 jahtml_DATA =  	gimageview.css 	index.html 	01.html 	02.html 	03.html 	04.html 	05.html 	06.html 	07.html 	gimv_faq.html 	gimv_tips.html 	gimv_history.html 	todo.html 	document.html 	changelog.html
 
 
-jatextdir = $(DESTDIR)$(gimv_docdir)/text/ja
+jatextdir = $(gimv_docdir)/text/ja
 jatext_DATA =  	COPYING.ja 	ChangeLog.ja 	README.ja 	INSTALL.ja 	NEWS.ja 	TODO.ja 	gimageview.txt
 
 
